clojure-mode.el

```diff
@@ -1009,7 +1009,7 @@ nil."
                     (one-or-more (any whitespace "\n")))
       ;; why is this here? oh (in-ns 'foo) or (ns+ :user)
       (zero-or-one (any ":'"))
-      (group (one-or-more (not (any "()\"" whitespace))) word-end)))
+      (group (one-or-more (not (any "()\"" whitespace))) symbol-end)))
```

Comment:
> AFAICT, that regexp should be just (1+ (any symbol word)). 

Correction, what I wrote is not valid rx. It would have to be `(1+ (or (syntax word) (syntax symbol)))`